src/high-level/double-float-matrix.lisp

```diff
@@ -0,0 +1,43 @@
+(in-package #:magicl)
+
+(defmatrix matrix/double-float double-float 0.0)
+
+(defmethod @ ((a matrix/double-float) (b matrix/double-float) &key target (alpha 1.0) (beta 0.0) transa transb)
+    "Multiply matrix a by matrix b, storing in target or creating a new matrix if target is not specified.
+Target cannot be the same as a or b."
+  ;; (check-type target (or matrix/double-float nil))
+  (check-type transa (member nil :n :t :c))
+  (check-type transb (member nil :n :t :c))
+  ;; TODO: make sure we can trans mixed order matrices (hard)
+  ;; Maybe just error with :row-major if :transX is :t or :c
+  (let* ((transa (or transa
+                     (if (eq :row-major (order a)) :t :n)))
```

Comment:
Maybe define an inlinable function that returns `(if (eq :row-major (order a)) "T" "N"))` and use that?